client/locations-list-map.js

```diff
@@ -231,10 +232,12 @@ function createFilters(data) {
     },
   };
 
+  // All states are included as potential filters through query and not selectable.
   for (const state of Object.keys(data)) {
     filters.states[state] = { name: state, isSet: false };
   }
 
+  // Load all other possible whitelisted filters with their accepted, translated values
```

Comment:
whitelisted -> allowlisted

As an industry we should prefer allowlist/blocklist over whitelist/blacklist going forward.